src/functions.js

```diff
@@ -36,40 +62,75 @@ function cities( capitals, formatter ){
   // such as 'Paris is the capital of France'
 
   // apply formatter to each object in capitals array and return an array of resulting sentences
+
+var mappedArray = capitals.map(function formatter(myArrItems) {
+      return (myArrItems.city + " is the capital of " + myArrItems.country)});
```

Comment:
please put the last `});` in a new line, as is not really clear what's going here otherwise

Reply:
thank you, I will.